docs/Validation-and-Serialization.md

```diff
@@ -313,7 +313,7 @@ fastify.post('/the/url', {
     }).required()
   },
   validatorCompiler: ({ schema, method, url, httpPart }) => {
-    return (data) => Joi.validate(data, schema)
+    return data => schema.validate(data)
```

Comment:
You should update the test accordingly

https://github.com/fastify/fastify/blob/2e9994d1c3cb0c61424d624674ec517360542c26/test/schema-examples.test.js#L221

Reply:
good catch